frontend/simple/js/state.js

```diff
@@ -88,7 +88,12 @@ const mutations = {
 // https://vuex.vuejs.org/en/modules.html
 const getters = {
   currentGroup (state) {
-    return state[state.currentGroupId]
+    const id = state.currentGroupId
+
+    if (!state[id]) return null
+
+    // Decorate currentGroup with id: state.currentGroupId
+    return { ...state[id], id }
```

Comment:
To keep the line count down, can you convert this into a 1-liner?

i.e. `return state[id] ? { ...state[id], id } : null`